api-reference/privacy/delete_request.mdx

```diff
@@ -7,15 +7,17 @@ You can delete traces data for a specific user of yours by specifying their asso
 
 ## Request Body
 
-<ParamField body="associationProperty" type="JSON">
-  A specific criterion for deletion like `{userId: "123"}`.
+<ParamField body="associationProperties" type="JSON">
```

Comment:
is JSON still the right type for a list?

Reply:
Yeah, mintlify still consider this a JSON since it's an array of json objects.